validator_derive/Cargo.toml

```diff
@@ -17,6 +17,6 @@ proc-macro = true
 syn = "2"
 quote = "1"
 proc-macro2 = "1"
-proc-macro-error = "1"
+proc-macro-error2 = "2"
```

Comment:
Maybe consider adding a "nightly" feature to the `validator_derive` crate?

Since [automatic nightly detection](https://github.com/GnomedDev/proc-macro-error-2/commit/e4e879d7def0b0a8910c7de0d7bdbedf8018795f) has been removed, and the "nightly" feature is disabled by default, some running with nightly compiler wont have nightly warnings.

Reply:
Thanks! Added

Reply:
You forget to add the nightly feature to the main crate too. Users wont use this crate directly but rather through the main `validator` crate.

Reply:
Yup oops